alica_engine/src/engine/allocationauthority/CycleManager.cpp

```diff
@@ -166,43 +166,43 @@ void CycleManager::setNewAllocDiff(const Assignment& oldAss, const Assignment& n
  */
 void CycleManager::handleAuthorityInfo(const AllocationAuthorityInfo& aai)
 {
-    if (!enabled) {
+    if (!_enabled) {
         return;
     }
     AgentIDConstPtr rid = aai.authority;
-    if (*rid == *myID) {
+    if (*rid == *_myID) {
```

Comment:
should be rid == _myId, AgentIDConstPtr has been newly introduced to facilitate that